settings.gradle

```diff
@@ -52,6 +52,12 @@ project(':nessie').name = 'iceberg-nessie'
 project(':gcp').name = 'iceberg-gcp'
 project(':dell').name = 'iceberg-dell'
 
+if (null != System.getProperty("allVersions")) {
+  System.setProperty("flinkVersions", System.getProperty("knownFlinkVersions"))
```

Comment:
If any of `flinkVersions` / `sparkVersions` / `hiveVersions` have been set in addition to `allVersions`, then I think it's probably ok to just overwrite whatever was set with all the versions, unless we'd want to support something like `./gradlew -DallVersions spotlessApply -DflinkVersions=xyz`  where `sparkVersions` / `hiveVersions` would be set to all, but `flinkVersions` to what was specified